conjur/controller/logout_controller.py

```diff
@@ -30,6 +34,10 @@ def remove_credentials(self):
         try:
             loaded_conjurrc = ConjurrcData.load_from_file(DEFAULT_CONFIG_FILE)
             if not self.credentials_provider.is_exists(loaded_conjurrc.conjur_url):
+                # Cleans up credentials leftover to make sure the environment is
```

Comment:
I think the following is a bit weird logically. Basically we are saying if something doesn’t exist then we want to cleanup.
```
if not self.credentials_provider.is_exists(loaded_conjurrc.conjur_url):
    self.logout_logic.cleanup_credentials(loaded_conjurrc)

def is_exists(self, conjurrc_conjur_url) -> bool:
    for attr in KEYSTORE_ATTRIBUTES:
         if KeystoreAdapter.get_password(conjurrc_conjur_url, attr) is None:
            return False
     return True
```

What about we do something else, and just cleanup **_all_** credentials on logout. The following in `logout_controller.py`
```
    def remove_credentials(self):
        try:
            self.logout_logic.cleanup_credentials(loaded_conjurrc)
   ...
```
Where `cleanup_credentials` in the `keystore_credentials_provider.py` looks like:
```
def cleanup_credentials(loaded_conjurrc)
  if not self.is_exists(loaded_conjurrc.conjur_url):
     raise Exception("You are already logged out.")
  for attr in KEYSTORE_ATTRIBUTES:
     try:
     ....
```
Where `is_exists`
```
    def is_exists(self, conjurrc_conjur_url) -> bool:
        for attr in KEYSTORE_ATTRIBUTES:
            if KeystoreAdapter.get_password(conjurrc_conjur_url, attr) is not None:
                return True
        return False
```